guppy/src/graph/build.rs

```diff
@@ -1061,4 +1054,49 @@ mod tests {
         // This should have forward slashes, even on Windows.
         assert_eq!(path.as_str(), "../../foo/bar/baz.txt");
     }
+
+    mod for_nextest_issue_1684 {
+        use super::convert_forward_slashes;
+        use crate::graph::build::Utf8Path;
+        #[track_caller]
+        fn verify_result_of_diff_utf8_paths(path_manifest: &str, path_workspace_root: &str, expected_relative_path: &str) {
+            let relative_path = pathdiff::diff_utf8_paths(
+                Utf8Path::new(path_manifest),
+                Utf8Path::new(path_workspace_root),
+            ).unwrap();
+            assert_eq!(convert_forward_slashes(relative_path), expected_relative_path);
+        }
+
+        #[test]
+        fn test_nextest_issue_1684_workspace_path_out_of_pocket() {
```

Comment:
No need for the issue number -- just `test_workspace_path_out_of_pocket` would work.